code/datums/traits/negative.dm

```diff
@@ -228,6 +161,306 @@
+			if(prob(0.5))
+				heirloom_addition += /obj/item/clothing/under/color/grey/glorf //very rare chance for ancient jumpsuit
+				//the actual chance is likely 0.083% or less.
+		if(JOB_STRING_JANITOR)
+			DEPARTMENT_FLAG |= DEPARTMENT_SERVICE
+			heirloom_addition += /obj/item/mop
+			heirloom_addition += /obj/item/clothing/suit/caution
+			heirloom_addition += /obj/item/reagent_containers/glass/bucket
+		if(JOB_STRING_BARTENDER)
+			DEPARTMENT_FLAG |= DEPARTMENT_SERVICE
+			heirloom_addition += /obj/item/reagent_containers/glass/rag
+			heirloom_addition += /obj/item/clothing/head/that
+			heirloom_addition += /obj/item/reagent_containers/food/drinks/shaker
+		if(JOB_STRING_COOK)
+			DEPARTMENT_FLAG |= DEPARTMENT_SERVICE
+			heirloom_addition += /obj/item/reagent_containers/food/condiment/saltshaker
+			heirloom_addition += /obj/item/kitchen/rollingpin
+			heirloom_addition += /obj/item/clothing/head/chefhat
+		if(JOB_STRING_BOTANIST)
+			DEPARTMENT_FLAG |= DEPARTMENT_SERVICE
+			heirloom_addition += /obj/item/cultivator
+			heirloom_addition += /obj/item/reagent_containers/glass/bucket
+			heirloom_addition += /obj/item/storage/bag/plants
+			heirloom_addition += /obj/item/toy/plush/beeplushie
+			heirloom_addition += /obj/item/seeds/random //Would you dare to plant your heirloom?
+		if(JOB_STRING_CURATOR)
+			DEPARTMENT_FLAG |= DEPARTMENT_SERVICE
+			heirloom_addition += /obj/item/pen/fountain
+			heirloom_addition += /obj/item/storage/pill_bottle/dice
+		if(JOB_STRING_CHAPLAIN)
+			DEPARTMENT_FLAG |= DEPARTMENT_SERVICE
+			heirloom_addition += /obj/item/toy/windupToolbox
+			heirloom_addition += /obj/item/reagent_containers/food/drinks/bottle/holywater
+		if(JOB_STRING_BARBER)
+			DEPARTMENT_FLAG |= DEPARTMENT_SERVICE
+			heirloom_addition += /obj/item/handmirror
+		if(JOB_STRING_VIP)
+			DEPARTMENT_FLAG |= DEPARTMENT_COMMAND //They'll just get more annoying items
+			DEPARTMENT_FLAG |= DEPARTMENT_SERVICE
+			heirloom_addition += /obj/item/handmirror //so narcissistic
+			heirloom_addition += /obj/item/modular_computer/laptop/preset/civillian //for business.
+		if(JOB_STRING_DEBTOR)
+			DEPARTMENT_FLAG |= DEPARTMENT_SERVICE
+			heirloom_addition += pick(random_trashes) //even such thing is precious to hobo. oh, poor.
+			heirloom_addition += pick(random_trashes)
+			heirloom_addition += pick(random_trashes)
+		if(JOB_STRING_LAWYER)
+			DEPARTMENT_FLAG |= DEPARTMENT_SERVICE
+			//DEPARTMENT_FLAG |= DEPARTMENT_SECURITY //maybe not...
+			heirloom_addition += /obj/item/gavelhammer
+			heirloom_addition += /obj/item/book/manual/wiki/security_space_law
+		//Entertainers
+		if(JOB_STRING_CLOWN)
+			DEPARTMENT_FLAG |= DEPARTMENT_SERVICE
+			heirloom_addition += /obj/item/bikehorn/golden
+			heirloom_addition += /obj/item/bikehorn/golden
+			heirloom_addition += /obj/item/bikehorn/golden
+			heirloom_addition += /obj/item/bikehorn/golden
+			heirloom_addition += /obj/item/bikehorn/golden	//high chance of spawning them
```

Comment:
you should make this a for loop, same with the other ones where you copy it 20 times

Reply:
they're now list. thanks.